src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -200,16 +220,16 @@ public ImageProcessor(ImageClassificationTransformer transformer)
                 _imagePreprocessingRunner.AddOutputs(transformer._resizedImageTensorName);
             }
 
-            public Tensor ProcessImage(string path)
+            public Tensor ProcessImage(VBuffer<byte> imgBuf)
             {
-                var imageTensor = new Tensor(File.ReadAllBytes(path), TF_DataType.TF_STRING);
+                var imageTensor = Encode(imgBuf);
```

Comment:
>Encode [](start = 34, length = 6)

Lets give it a more meaningful name like EncodeByteAsString.

Reply:
Done.